src/redux/Category/Category.js

```diff
@@ -0,0 +1,19 @@
+const CHECK_STATUS = 'bookstore-react-app/Category/CHECK_STATUS';
+
+const CategoryReducer = (state = [], action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return action.text;
+    default:
+      return state;
+  }
+};
+
+export const checkStatus = (text) => {
+  const status = {
+    type: CHECK_STATUS,
+    text,
+  };
+  return status;
+};
```

Comment:
- [ ] Here, the `checkStatus` method is expecting a text as param.  But when you are calling this action, you are not passing a text, this will lead to errors:

```
#bookstore/src/components/Category.js
<button type="button" onClick={() => dispatch(checkStatus())}>Check status</button>
```

Please make sure that all functions work properly.